src/amo/components/Addon/index.js

```diff
@@ -274,28 +273,23 @@ export class AddonBase extends React.Component {
           title = i18n.gettext('About this add-on');
       }
 
-      const description = addon.description ? addon.description : addon.summary;
-      // For any theme type, we want to hide the summary text here since that is
-      // already displayed in the header.
-      showAbout =
-        ((description === addon.summary &&
-          addon.type === ADDON_TYPE_STATIC_THEME) ||
-          addon.type === ADDON_TYPE_THEME) === false;
```

Comment:
Hey @SeanPrashad, thanks for helping with this one : ) 

So I think we still need something like we had above. I know it gets a little confusing b/c the different types have different fields. This should be the breakdown:

Extensions,
Static themes:

Description - yes (optional)
Summary - yes (required on both)
-----

Lwt themes:
Description - yes (optional)
Summary - always null

You can inspect this in the redux tool too if that helps.


so I *think* you can just revert a lot of what was here and just remove static theme piece ~:

```javascript
const description = addon.description ? addon.description : addon.summary;

showAbout =
        (description === addon.summary || addon.type === ADDON_TYPE_THEME) ===
        false;
...
```

Basically for a lightweight theme we never want to show that bottom card (it will never have new/different text). For other types we just need to see if it's actually using the `addon.description` text. If the "description" here is set to the `addon.description` we know the addon has both fields (since summary is required) which means we should display both texts. If the "description" is set to the `addon.summary` then we can hide in the bottom card since since we are already displaying this text in the header. 

note: I'm just thinking it might be a *little* less confusing if we renamed `const description = ...` to `const text=...` since it could really either be addon.description or addon.summary?


Does that help? or let me know if any questions :)

Reply:
Thanks for the super detailed explanation @rebmullin! It makes more sense now :)